inlong-sort/sort-formats/format-json/src/main/java/org/apache/inlong/sort/formats/json/debezium/DebeziumJsonDeserializationSchema.java

```diff
@@ -195,7 +210,10 @@ public void deserialize(byte[] message, Collector<RowData> out) throws IOExcepti
     }
 
     private void emitRow(GenericRowData rootRow, GenericRowData physicalRow, Collector<RowData> out) {
-        final int physicalArity = physicalRow.getArity();
+        int physicalArity = physicalRow.getArity();
+        if (isMigrateAll) {
+            physicalArity -= 1;
```

Comment:
if isMigrateAll, physical arity should always be 0. because all fields are treated as metadata fields including the `MYSQL_METADATA_DATA` field

what's the point of physicalArity -= 1 ?